FAQ.md

```diff
@@ -31,13 +42,34 @@ Supernode will maintain a bitmap which records the correspondence between peers
 
 **NOTE**: The scheduler will decide whether to download from the supernode or other peers. As for the detail of the scheduler, please refer to [scheduler algorithm](#what-is-the-peer-scheduling-algorithm-by-default)
 
+## What is the size of block(piece) when distribution
+
+Dragonfly tries to make block(piece) size dynamically to ensure efficiency.
+
+The size of pieces which is calculated as per the following strategy:
```

Comment:
`The size of pieces which is calculated as per the following strategy:`->`The size of pieces  is calculated the following strategy:`